index.js

```diff
@@ -83,9 +87,15 @@
         }, $this.data('load-after'));
       });
     } else {
-      whenScroll(['within 300px of', $newElement[0]], function () {
-        loadNow($this, $newElement);
-      }, true);
+      // This is essential for firing the event to a DOM immediately, as set by
```

Comment:
We all love the mysterious third parameter. Probably could be updated in when-scroll, yeah. Want me to switch it over? If so, can you add me as a collab on the repo?